lib/types/utils.ts

```diff
@@ -60,12 +60,35 @@ export interface EntrySys extends ContentEntitySys {
   automationTags: Link<'Tag'>[]
 }
 
-export interface Items {
-  type: string
-  linkType?: string
+export type FieldType =
+  | 'Symbol'
+  | 'Text'
+  | 'RichText'
+  | 'Number'
+  | 'Integer'
+  | 'Array'
+  | 'Link'
+  | 'Object'
+  | 'Date'
+  | 'Location'
```

Comment:
Isn't the type Boolean necessary here?

Reply:
Good catch! I will add it.

Reply:
https://github.com/contentful/ui-extensions-sdk/pull/1541